### DISCUSSION TITLE
Update the Link Component so it can open screens in Modal

### DISCUSSION THREAD

**Discussion by EuanRW** [Budibase/budibase#7228] [Ideas]
Currently, I can only open screens from a link component as a whole page or in a new tab, I would like to be able to open modal screens from the link component like I can from the button or icon component.

**Comment by jaz000** [comment]
+1 yes please, just came to look for the same question
where can I open a modal from? Why only some places?

You can do it from a button - it'll probably look similar ... ish

**Reply by EuanRW** [reply]
Thanks Jaz,

The work around I am using at the moment is to use a global styling embed to style the "over background" button to look more like a link:

```
<style>  

.spectrum-Button--overBackground:hover {  
    color: blue !important;  
   font-weight: normal !important;  
}  
.spectrum-Button--overBackground {  
       color: black !important;;  
      font-weight: normal !important;  
}  

</style> 
```

https://docs.budibase.com/docs/custom-css#global-styling

**Reply by melohagan** [reply]
A quiet button may also suffice as it removes the border:
![Screenshot 2022-08-16 at 10 07 10](https://user-images.githubusercontent.com/101575380/184842106-ee6039e8-cb88-4c6d-a338-b2033b1a5c16.png)
![Screenshot 2022-08-16 at 10 07 26](https://user-images.githubusercontent.com/101575380/184842156-f85f2333-ff3a-4dd5-b1da-6f0485a71b0a.png)

Otherwise the embed component styling could be an option. 

In my opinion, a link shouldn't open a modal. From a UX perspective opening a modal is an action, you are not navigating - you are staying on the same page.
But I'm also not a designer so what do I know 😁
